examples/save_load.rs

```diff
@@ -17,7 +17,19 @@
 
 pub mod shared;
 
-use crate::shared::{create_ui, fix_shadows_distance, Game, GameScene, LocomotionMachine, Player};
+use crate::shared::{
```

Comment:
Please make a separate example for custom resource loaders. `save_load` example is a about serialization, not about resources.